test/it/parser/src/main/resources/case/dml/select-special-function.xml

```diff
@@ -2644,4 +2644,144 @@
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_found_rows">
+        <projections start-index="7" stop-index="18">
+            <expression-projection start-index="7" stop-index="18" text="FOUND_ROWS()">
+                <expr>
+                    <function function-name="FOUND_ROWS" text="FOUND_ROWS()" start-index="7" stop-index="18">
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_from_base64">
+        <projections start-index="7" stop-index="35">
+            <expression-projection start-index="7" stop-index="35" text="FROM_BASE64(TO_BASE64('abc'))">
+                <expr>
+                    <function function-name="FROM_BASE64" text="FROM_BASE64(TO_BASE64('abc'))" start-index="7" stop-index="35">
+                        <parameter>
+                            <function function-name="TO_BASE64" text="TO_BASE64('abc')" start-index="19" stop-index="34">
+                                <parameter>
+                                    <literal-expression value="abc" start-index="29" stop-index="33" />
+                                </parameter>
+                            </function>
+                        </parameter>
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_from_days">
+        <projections start-index="7" stop-index="23">
+            <expression-projection start-index="7" stop-index="23" text="FROM_DAYS(730669)">
+                <expr>
+                    <function function-name="FROM_DAYS" text="FROM_DAYS(730669)" start-index="7" stop-index="23">
+                        <parameter>
+                            <literal-expression value="730669" start-index="17" stop-index="22" />
+                        </parameter>
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_from_unixtime">
+        <projections start-index="7" stop-index="31">
+            <expression-projection start-index="7" stop-index="31" text="FROM_UNIXTIME(1447430881)">
+                <expr>
+                    <function function-name="FROM_UNIXTIME" text="FROM_UNIXTIME(1447430881)" start-index="7" stop-index="31">
+                        <parameter>
+                            <literal-expression value="1447430881" start-index="21" stop-index="30" />
+                        </parameter>
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_get_format">
+        <projections start-index="7" stop-index="29">
+            <expression-projection start-index="7" stop-index="29" text="GET_FORMAT(DATE, 'EUR')" />
```

Comment:
Why this case not parse parameters?

Reply:
when I use function tag to parse it, I encountered an error that the actal parameters not match the expected parameters, because of DATE was parsed as Column rather than Type.

Reply:
when I used the function tag to parse it, I encountered an error that the actual parameters did not match the expected parameters, because DATE was parsed as Column rather than Type.  
but 'EUR' is also a parameter that should be set. I'll fix it.